docs/content/en/_index.md

```diff
@@ -8,15 +8,21 @@ menu:
     weight: 20
   footer:
     weight: 60
-description: "K8ssandra documentation: architecture, configuration, guided tasks"
+description: "K8ssandra documentation: architecture, configuration, guided tasks."
 type: docs
 ---
 
 The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. Check out the sections at left based on your needs and be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
 
-{{% alert title="Tip" color="primary" %}}
-If you're impatient, jump right in with our **[install]({{< relref "install" >}})** topics!
-{{% /alert %}}
+**Tip:** Starting with the March 2022 introduction of K8ssandra Operator, the K8ssandra Operator custom resource, and its support for single- or **multi-cluster** deployments, the documentation site has been organized into three areas. 
+
+* [docs.k8ssandra.io](https://docs-staging.k8ssandra.io/) provides topics that are of common interest to users of K8ssandra Operator and K8ssandra, such as FAQs, Release Notes, Components, and a Glossary.
+* [docs-v1.k8ssandra.io](https://docs-staging-v1.k8ssandra.io/) (**you are here**) provides topics that are specific to K8ssandra 1.4.x users (the initial project releases).
```

Comment:
Should we just put `1.x` here instead of `1.4.x`?  We can't say for certain there will not be another `1.5.0` release -- whereas I do think we're highly confident that when we get to a point that the k8ssandra/k8ssandra charts use the k8ssandra-operator that will be a 2.0.0 release of the charts.

This isn't minor at the moment, because it is definitely correct as it is, just don't want you to have to come back and update that with other releases.

Reply:
I hear ya. I'll keep it this way for now. Don't want some users to confuse "1.x" in K8ssandra <version> and K8ssandra Operator v1.x.x.